languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -69549,7 +69549,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <example>mais ce n'est pas avant l'année 1847</example>
                 <example>Meilleurs vœux à Tom et Marie pour l'année 2018.</example>
             </rule>
-            <rule>
+            <rule default="off">
```

Comment:
suggestion: go futher and remove the rule. We won't improve that one and it would just rejoin the ranks of off rules we try to get rid off in https://github.com/languagetooler-gmbh/languagetool-premium/issues/5596 🥜

Reply:
Right! I'll merge and then I'll delete the rule in my next commit :)